.nvmrc

```
@@ -0,0 +1 @@
+v20.11.1
```

To match the `.node-version` file

```suggestion
v18.19.0
```

https://github.com/ChildMindInstitute/mindlogger-admin/blob/57051d6026285cb4433900db512861886524f04e/.node-version#L1